app/models/task.py

```diff
@@ -1,6 +1,24 @@
 from flask import current_app
 from app import db
+from sqlalchemy import DateTime
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+
+    def to_json(self):
+        task = {
+            "id": self.task_id,
+            "title": self.title,
+            "description": self.description,
+            "is_complete": False if self.completed_at is None else True
```

Comment:
Nice ternary! Another way to write this line:

```suggestion
            "is_complete": self.completed_at != None
```